.github/workflows/ADD_TO_HTO_PROJECT.yml

```diff
@@ -0,0 +1,20 @@
+name: ADD_TO_PROJECT
+
+on:
+  issues:
+    types:
+      - opened
+      - transferred
+  pull_request:
+    types:
+      - opened
+
+jobs:
+  Exec:
+    name: Add issue to project
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/add-to-project@d3b6ae9953cfef24c2e92848b832e7c91d6dba67
+        with:
+          project-url: ${{ secrets.HTO_PROJECT_URL }}
+          github-token: ${{ secrets.ADD_TO_PROJECT_PAT }}
```

Comment:
[We could add an empty new line at the end of file so it's POSIX compliant and it doesn't break stuff like `cat`](https://stackoverflow.com/a/729795)

Reply:
Makes totally sense, I just added it 👍